stable/enterprise/templates/_common.tpl

```diff
@@ -247,6 +247,53 @@ app.kubernetes.io/component: {{ $component | lower }}
 {{- end -}}
 
 
+{{/*
+Create an image specification template that can override the default image
+based on global settings
+*/}}
+{{- define "enterprise.common.imageOverride" -}}
```

Comment:
"Override" is confusing here, it should just be "enterprise.common.image".

Reply:
Ack, so should it be:

image: docker.io/anchore/enterprise:v5.16.0
  # registry: docker.io
  # repository: anchore/enterprise
  # tag: v5.16.0
  # digest: sha256:12345abcde
  
Then if registry / repo / tag / digest are defined, use that, otherwise default to just the pullstring?

or should I split it out into something like:

image: 
  pullstring: docker.io/anchore/enterprise:v5.16.0
  # registry: docker.io
  # repository: anchore/enterprise
  # tag: v5.16.0
  # digest: sha256:12345abcde
  
And then do the same checks but modify .Values.image to .Values.image.pullstring?

Reply:
Disregard, saw your later comment about checking the 'image' value either being a map or a string